static/collection.js

```diff
@@ -0,0 +1,42 @@
+(async () => {
+async function create_all_games_table() {
+    const user = document.getElementById("user").value;
+    fetch('/all_games?user=' + user)
+        .then(async response => {
+            if (response.ok) {
+                const games = await response.json();
+
+                const table_element = document.getElementById("table");
+                let table = '';
```

Comment:
using a string to build the HTML certainly works, but i think it's a little trickier to read and harder to maintain (not to mention it might not be as performant.) also, if the BGG Companion API ever returned a response that contains XSS, your site _might_ be vulnerable to that since you use `innerHTML` later in the function.

the `document` object in JavaScript has a [`createElement`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) method that you can use to create arbitrary DOM objects and add them to a parent node. this might be a better approach, definitely more work but i'd be happy to pair with you on this!